examples/games/alien_cake_addict.rs

```diff
@@ -105,6 +108,8 @@ fn setup_cameras(mut commands: Commands, mut game: ResMut<Game>) {
 }
 
 fn setup(mut commands: Commands, asset_server: Res<AssetServer>, mut game: ResMut<Game>) {
+    let mut seeded_rng = StdRng::seed_from_u64(19878367467713);
```

Comment:
```suggestion
    // Make the game play out the same way every time. We do this for testing purposes.
    // If you want the game to play out differently every time, use `StdRng::from_entropy()`.
    let mut seeded_rng = StdRng::seed_from_u64(19878367467713);
```

Or something like that? A hardcoded seed doesn't really make sense for gameplay, so I feel like in this "game example" we should probably explain ourselves.

Reply:
using a seeded random does make sense for gameplay, for example if you want to be able to do speed run and compare between them

Reply:
I absolutely agree that seeded random numbers have a place in games, but would argue that as the default mode for this game it would be surprising behavior. I suppose that's a matter of taste, though.

I think if we are doing this specifically to make CI deterministic, we should at least tell users why. I would actually prefer a comment to code for changing the behavior in CI which just seems like an additional distraction for users trying to learn from the examples.

(Non-blocking commentary.)